qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
Just for knowledge. So this change will go to unreleased branch and later on this will be merge to master. So for unreleased branch it will affect quality team only but once it is merge to master it will be used by everyone ?